components/central-application-gateway/README.md

```diff
@@ -0,0 +1,109 @@
+# Central Application Gateway
+
+## Overview
+
+This is the repository for the Central Application Gateway.
+
+## Prerequisites
+
+The Central Application Gateway requires Go 1.8 or higher.
+
+## Installation
+
+To install the Central Application Gateway, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+2. `cd kyma/components/central-application-gateway`
+3. `CGO_ENABLED=0 go build ./cmd/applicationgateway`
+
+## Usage
+
+This section explains how to use the Central Application Gateway.
+
+### Start the Central Application Gateway
+
+To start the Central Application Gateway, run this command:
+
+```
+./applicationgateway
+```
+
+The Central Application Gateway has the following parameters:
+- **disableLegacyConnectivity** is the flag for disabling the default legacy and enabling the Compass mode. The default value is `false`.
+- **proxyPort** is the port that acts as a proxy for the calls from services and Functions to an external solution. The default port is `8080`.
+- **externalAPIPort** is the port that exposes the API allowing to check component status. The default port is `8081`.
+- **application** is the Application name used to write and read information about services. The default Application is `default-ec`.
+- **namespace** is the Namespace in which the Central Application Gateway is deployed. The default Namespace is `kyma-system`.
+- **requestTimeout** is the timeout for requests sent through the Central Application Gateway, expressed in seconds. The default value is `1`.
+- **skipVerify** is the flag for skipping the verification of certificates for the proxy targets. The default value is `false`.
+- **requestLogging** is the flag for logging incoming requests. The default value is `false`.
+- **proxyTimeout** is the timeout for requests sent through the proxy, expressed in seconds. The default value is `10`.
+- **proxyCacheTTL** is the time to live of the remote API information stored in the proxy cache, expressed in seconds. The default value is `120`.
+
+
+## API
+The Central Application Gateway exposes:
+- external API implementing health endpoint for liveness and readiness probes
+- internal API implementing proxy handler accessible via a service of ClusterIP type
+
+### Standalone mode
+In case  **disableLegacyConnectivity** is `false` the proxy API exposes the following endpoint:
+```{Application name}/{Service name}/{target API path}``` 
+
+For instance, if the user registered `cc-occ-commerce-webservices` service in `ec` application using Application Registry, they can send a request to the following url:
+```http://central-application-gateway:8080/ec/cc-occ-commerce-webservices/basesites```
```

Comment:
``````suggestion
For instance, if the user registered the `cc-occ-commerce-webservices` service in the `ec` application using Application Registry, they can send a request to the following URL: 
```bash
http://central-application-gateway:8080/ec/cc-occ-commerce-webservices/basesites`
```
``````

Added the definite articles `the` in front of the names of the application and service, and fixed the formatting of the URL block.
<br> 
Also, shouldn't the application name here be `default-ec`, and not just `ec`? You give it as a default in line 35.